classes/statistics/PKPStatisticsHelper.php

```diff
@@ -305,7 +305,7 @@ public function getInstitutionIds(int $contextId, string $ip, string $hashedIp,
 
         if ($flush) {
             // Salt and thus hashed IPs changed, empty the cache.
-            $this->institutionDataCacheMiss($this->institutionDataCache, 'ID');
+            $this->institutionDataCacheMiss($this->institutionDataCache);
```

Comment:
Merged this commit -- but heads-up @bozana just in case.

Reply:
All fine! :+1: Thanks!